Empire/Empire.cpp

```diff
@@ -1458,6 +1498,8 @@ int Empire::StockpileID(ResourceType res) const {
         return m_capital_id;
         break;
     case RE_INDUSTRY:
+        return IMPERIAL_OBJECT_ID;
```

Comment:
So, did I read you right? I should remove all the code relating to stockpile locations?